### PR TITLE
Set default max container root partition size

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-2075

Preview build: https://deploy-preview-32774--osdocs.netlify.app/

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@umohnani8 PTAL Thank you!

File: modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+----
+$ oc get machineconfigs
+----
++
+You will see a new `99-worker-zxy-containerruntime` object as well as a new `rendered-worker-xyz` object created:
++
+.Example output
+[source,terminal]
+----
+99-worker-xyz-containerruntime   4173030d89fbf4a7a0976d1665491a4d9a6e54f1   2.2.0             7m42s
```
Comment (umohnani8) [inline modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc:70]: The MC names changed some time back, this should be `99-worker-generated-containerruntime` now.

**Comment by umohnani8** [comment]
LGTM with one change request

File: modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc]: Updated. Thank you!

**Comment by ahardin-rh** [comment]
@sunilcio Can you please provide QE review? Thank you!

File: modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+----
++
+When complete, the worker nodes will transition back to `UPDATING` as `False` with the number of `UPDATED` matching the `MACHINECOUNT`:
++
+.Example output
+[source,terminal]
+----
+NAME   CONFIG              UPDATED   UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+worker   rendered-worker-xyz   True      False      False      3         3            3             0           20h
+----
```
Comment (lbarbeevargas) [inline modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc:97]: I think the alignment of the output columns might be off here as well.

File: modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
++
+Looking at a worker machine, you see the new 8GB max size configuration is applied to all of the workers:
```
Comment (lbarbeevargas) [inline modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc:99]: Small nit, and for localization best practice, you could add "that" here. For example:

> Looking at a worker machine, you see that the new 8 GB max size configuration is applied to all of the workers:

Looks like ISG wants a space before GB as well.

File: modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+[storage]
+  driver = "overlay"
+  runroot = "/var/run/containers/storage"
+  graphroot = "/var/lib/containers/storage"
+  [storage.options]
+    additionalimagestores = []
+    size = "8G"
+----
++
+Looking inside a container, you see the root partition is now 8GB:
```
Comment (lbarbeevargas) [inline modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc:114]: Similar small nit to add "that" and a space before GB:

> Looking inside a container, you see that the root partition is now 8 GB:

**Review by lbarbeevargas — COMMENTED** [review]
Looks great! Just a few wording and ISG considerations.

**Comment by sunilcio** [comment]
@lyman9966 could you help review this, thanks!

**Comment by ahardin-rh** [comment]
Looks like post_installation_configuration/machine-configuration-tasks.adoc  does not exist in 4.5, so we'll skip the cherrypick to that version

**Pull request merged**